gp-address-autocomplete/gpaa-single-line-as-input-value copy.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Gravity Perks // GP Address Autocomplete // Use Single Line Text field as autocomplete input
+ * https://gravitywiz.com/documentation/gravity-forms-address-autocomplete
+ *
+ * Instruction Video: https://www.loom.com/share/680513c2f0dc403aa6a001f9950f4e77
+ *
+ * This snippet allows you to use Single Line Text fields as input selectors for GP Address Autocomplete instead of the default Address field.
+ *
+ * Description:  Customize the filter name and autocomplete input selector accordingly
+ * Author:       Gravity Wiz
+ * Version:      0.1
+ * Author URI:   https://gravitywiz.com
```

Comment:
This can be removed. It appears to be the old format we used for snippet headers.

Latest deets here: https://www.notion.so/gravitywiz/Snippet-Library-Code-Organization-Formatting-and-Naming-d5bee101d9674dfc98d7ec007867de2b